app/src/features/projects/components/ProjectDetailsForm.tsx

```diff
@@ -132,6 +132,7 @@ const ProjectDetailsForm: React.FC<IProjectDetailsFormProps> = (props) => {
               variant="outlined"
               value={values.end_date}
               type="date"
+              inputProps={{ min: '1900-01-01', max: '2100-12-31' }}
```

Comment:
Can you add these dates to /constants/dateFormat.ts and import them into the files that use them?

Reply:
Added!